examples/simple_trace_report.rs

```diff
@@ -1,14 +1,67 @@
+use std::error::Error;
```

Comment:
Could we add new file which includes self-contained example?

Reply:
Yes. Let me try to build an e2e test for it.